doc/dev/mempool_accounting.rst

```diff
@@ -0,0 +1,56 @@
+What is a mempool?
+------------------
+A memory pool is a method for accounting the consumption of memory of
+a set of containers.
```

Comment:
We have mempools without containers too, suggest rewording to "Memory pools (mempools) show the details of memory consumption by each Ceph daemon".

Reply:
@anthonyeleven good that we can assess what idea it conveys, the actual meaning here is memory accounting of C++ classes and containers used in mostly bluestore component. Perhaps,

```suggestion
Memory pools (mempools) show the details of memory consumption of C++ classes and containers, which can then be used to assess memory leaks, and other insights around memory usage with low overhead.
```